test/widget/section.test.ts

```diff
@@ -0,0 +1,45 @@
+import { SectionWidget } from '../../src/widget/section';
+
+test('snapshot test: minimum properties', () => {
```

Comment:
Since these are not snapshot tests, let's rename it accordingly. Same for the next test.

Reply:
My miss, will fix.